### DISCUSSION TITLE
Ignition guide for creating UUPS upgradeable module

### DISCUSSION THREAD

**Discussion by bomb-on** [NomicFoundation/hardhat#5529] [General]
Any chance we get a guide about how to create an Ignition module(s) for OpenZeppelin's UUPS upgradeable contracts? Something similar to the [Transparent Upgradeable guide](https://hardhat.org/ignition/docs/guides/upgradeable-proxies)?
Thanks!

**Comment by kanej** [comment]
Hey @bomb-on, that is a great suggestion. I have opened an issue in the Ignition repo: https://github.com/NomicFoundation/hardhat-ignition/issues/788

**Reply by bomb-on** [reply]
Thank you very much, looking forward to see this guide! :)
